packages/core/src/render3/instructions.ts

```diff
@@ -799,6 +797,9 @@ export function listener(
     eventName: string, listenerFn: (e?: any) => any, useCapture = false): void {
   const lView = getLView();
   const tNode = getPreviousOrParentTNode();
+  const tView = lView[TVIEW];
+  const firstTemplatePass = tView.firstTemplatePass;
+  const tCleanup: false|any[] = firstTemplatePass && (tView.cleanup || (tView.cleanup = []));
```

Comment:
```suggestion
  const tCleanup: false|any[] = firstTemplatePass && getTViewCleanup(lView);
```

We could then get rid of the `tView` local var.

Reply:
can't get rid of `tView` as it is used above.  This change was intentional as I want to minimize the global reads.